helm/kubernetes-nginx-ingress-controller-chart/templates/controller-deployment.yaml

```diff
@@ -95,7 +95,7 @@ spec:
             exec:
               command:
               - sleep
-              - "15"
+              - "60"
```

Comment:
15 seconds seemed short for me. I saw other charts with HPA use 60 seconds, so I adapted this.

Reply:
cool, this makes it more graceful